src/main/java/com/yahoo/bullet/bql/tree/ManualDistributionNode.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright 2018, Oath Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.bql.tree;
+
+import com.yahoo.bullet.aggregations.Distribution.Type;
+import lombok.Getter;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static com.yahoo.bullet.aggregations.Distribution.POINTS;
+import static com.yahoo.bullet.aggregations.Distribution.TYPE;
+import static java.util.Objects.requireNonNull;
+
+@Getter
+public class ManualDistributionNode extends DistributionNode {
+    private final List<Double> points;
+
+    /**
+     * Constructor that requires a List of {@link ExpressionNode} columns, a {@link Type} and a List of Double points.
+     *
+     * @param type    A {@link Type}.
```

Comment:
Is this style of indenting javadocs consistent?

Reply:
No clue but will figure it out. We don't technically need javadocs in bql, correct?

Reply:
Technically sure